frontend/src/packages/users/users-api.ts

```diff
@@ -29,6 +33,22 @@ class UserApi extends HttpApi {
 
     return await response.json<UserGetAllResponseDto>();
   }
+
+  public async updateUser(
+    payload: UserUpdateRequestDto,
+  ): Promise<UserAuthResponse> {
+    const response = await this.load(
+      this.getFullEndpoint(UsersApiPath.ROOT, {}),
+      {
+        method: 'POST',
```

Comment:
```suggestion
        method: 'PUT',
```

Reply:
"should be post on root" he said. Will fix it in a few minutes